CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.10.1 - 2017-04-11
+
+- Update fury to 3.0.0-beta.1
```

Comment:
This patch update would introduce a breaking change of dropping node 0.10/0.12 support?